web/src/app/Connection.js

```diff
@@ -63,7 +70,7 @@ class Connection {
         this.ws = null;
       } else {
         const retrySeconds = retryBackoffSeconds[Math.min(this.retryCount, retryBackoffSeconds.length - 1)];
-        this.retryCount++;
+        this.retryCount += 1;
```

Comment:
Can you elaborate on why this is better? i++ seems clearer to me and verrrry common.

Reply:
https://github.com/airbnb/javascript#variables--unary-increment-decrement

Reply:
Alright, I guess :-D